### PR TITLE
feat: cypress test runner

### PR DESCRIPTION
**Pull request by gabewomble**
This PR introduces the Cypress test runner to our application! It only has a basic configuration at this time. We will want to setup a github action to run these tests on PRs and merges to master, but the effort required warrants a separate PR.

Screen Shot 2021-06-04 at 3 04 53 PM

### CODE REVIEW

File: src/Header/index.tsx [hunk 1]
```diff
@@ -2,7 +2,15 @@ import { Box, Flex, Heading, Input, Text } from "@chakra-ui/react";
...
-    <Box borderRadius="md" w="100%" px={1} py={3} color="white" bg="purple.600">
+    <Box
+      bg="purple.600"
+      borderRadius="md"
+      color="white"
+      data-test="header"
+      px={1}
+      py={3}
+      w="100%"
+    >
```
Comment (MrArnoldPalmer) [inline src/Header/index.tsx:13]: Seems like this change is unrelated.

**Review by MrArnoldPalmer — CHANGES_REQUESTED** [review]
just removal of the change in the Header component then 👍🏻

File: src/Header/index.tsx (hunk 1 above)
Comment (gabewomble) [reply src/Header/index.tsx:13]: Only addition is the `data-test` attr which made the Header exceed our character limit for the line

**Review by MrArnoldPalmer — APPROVED** [review]
